### ISSUE TITLE
close one by one card type error message when using control shift c

### ISSUE THREAD

**Issue by dalton264** [AnKingMed/AnKing-Note-Types#106]
#### Problem description

When I click use the cloze one by one template sometimes when i use **(control + shift +c)** I get an error message. 

#### Checklist

*Please replace the space inside the brackets with an **x** if the following items apply:*

 - [ x] I've restarted Anki to see if it helps
 - [ x] I've verified that I use the latest version of the note type by redownloading it from AnkiWeb
 - [ x] I've tried to disable add-ons to see if there are any interactions present
 - [x ] My issue disappears when I hold shift while starting Anki.
 - [x ] I've checked if anyone else reported this problem before by looking through the issue reports. I also checked to see if there is a section about known issues in the add-on description, documentation, or README.
 

#### What Anki version are you using?

*Please open Anki, go to Help → About (windows) or Anki → About Anki (mac) to find the version. Edit the value between the backticks below

```
2.1.55
```
#### What Operating system are you using? (i.e. MacOS, Windows, Linux)

*Please include OS version as well if possible*
Windows

#### Error message (if any)

*If you've received an error message, please copy and paste it between the backticks below:*
Error
An error occurred. Please start Anki while holding down the shift key, which will temporarily disable the add-ons you have installed.
If the issue only occurs when add-ons are enabled, please use the Tools > Add-ons menu item to disable some add-ons and restart Anki, repeating until you discover the add-on that is causing the problem.
When you've discovered the add-on that is causing the problem, please report the issue to the add-on author.
Debug info:
Anki 2.1.55 (01caec2a) Python 3.9.15 Qt 6.4.0 PyQt 6.4.0
Platform: Windows-10-10.0.19045
Flags: frz=True ao=True sv=2
Add-ons, last update check: 2023-01-05 11:49:46

Caught exception:
Traceback (most recent call last):
  File "aqt.webview", line 43, in cmd
  File "aqt.webview", line 150, in _onCmd
  File "aqt.webview", line 614, in _onBridgeCmd
  File "_aqt.hooks", line 4887, in __call__
  File "C:\Users\dalto\AppData\Roaming\Anki2\addons21\272311064\src\webview\__init__.py", line 90, in add_occlusion_messages
    on_cloze(editor)
  File "C:\Users\dalto\AppData\Roaming\Anki2\addons21\272311064\src\editor\__init__.py", line 182, in on_cloze
    return _old(editor)
NameError: name '_old' is not defined

```

```

**Comment by AnKingMed** [comment]
It looks like this is an issue with the closet add-on. Is it up to date? Maybe try deleting and redownloading that add-on? I'm not having any issues with it

**Comment by dalton264** [comment]
> It looks like this is an issue with the closet add-on. Is it up to date? Maybe try deleting and redownloading that add-on? I'm not having any issues with it

I checked to make sure it was uptodate and even re-downloaded it but I still get the message; however, I do not receive an error message when I use the (contrl + alt + shift + c) command.

**Comment by AnKingMed** [comment]
I'm not having any issues. Can you test with just the clozet addon installed? It might be related to another add-on

**Comment by dalton264** [comment]
I disabled all addons and even reinstalled the closet addon just to be safe but I still got this error message.

**Comment by AnKingMed** [comment]
This only occurs when using the Cloze-one by one note type?

**Comment by dalton264** [comment]
Thats the only note type i tried at the time. I just quickly checked the AnKing and the overhaul note type and those both also had an error message. I think it is the same. 
Error
An error occurred. Please start Anki while holding down the shift key, which will temporarily disable the add-ons you have installed.
If the issue only occurs when add-ons are enabled, please use the Tools > Add-ons menu item to disable some add-ons and restart Anki, repeating until you discover the add-on that is causing the problem.
When you've discovered the add-on that is causing the problem, please report the issue to the add-on author.
Debug info:
Anki 2.1.55 (01caec2a) Python 3.9.15 Qt 6.4.0 PyQt 6.4.0
Platform: Windows-10-10.0.19045
Flags: frz=True ao=True sv=2
Add-ons, last update check: 2023-01-06 14:32:44

Caught exception:
Traceback (most recent call last):
  File "aqt.webview", line 43, in cmd
  File "aqt.webview", line 150, in _onCmd
  File "aqt.webview", line 614, in _onBridgeCmd
  File "_aqt.hooks", line 4887, in __call__
  File "C:\Users\dalto\AppData\Roaming\Anki2\addons21\272311064\src\webview\__init__.py", line 90, in add_occlusion_messages
    on_cloze(editor)
  File "C:\Users\dalto\AppData\Roaming\Anki2\addons21\272311064\src\editor\__init__.py", line 182, in on_cloze
    return _old(editor)
NameError: name '_old' is not defined

**Comment by AnKingMed** [comment]
Can you open a new profile and import the note types brand new using the AnKing Note Types add-on and see if you get the error code?

**Comment by dalton264** [comment]
i created a new profile and went to the anking note type addon where I imported the AnKing and overhaul deck but still no like. Just to clarify, I get this error message but I can still create the cards and can view them fine.

**Comment by AnKingMed** [comment]
This is weird. I can't reproduce the error at all. I wonder if it's a windows thing?

**Comment by dalton264** [comment]
Okay this is very odd. Just to check something I used the normal cloze deletion option on the new user and I noticed using the normal cloze deletion I also received this error message, but when I disabled both the closet addon and the anking note types I did not have the error message occur with a normal cloze note type. Additionally, I then left the anking note types addon enabled and disabled the closet addon and received no error message. So this would mean the problem is with the closet addon? Checking the addon website I see that it was updated for anki version 2.1.52-2.1.54+ (Updated 2022-08-12). 

Yeah, I am really not sure. I just tried checking my other windows 10 computer running anki 2.1.54 (b6a7760c) and it also had the same error message. However, when I disabled the closet addon I did not receive the error message on this computer either.
